feat: add rewards (integrator) doc (#1022)
axelarnetwork/axelar-docs: src/pages/dev/amplifier/add-rewards.mdx

## Check the funds in a pool

Use `rewards_pool()` to check the funds in a pool:

<tabs>

<tab-item title="<code>devnet-verifiers</code>">
    ```bash
    export REWARDS_CONTRACT_ADDRESS="axelar1guczj53xxl4347adagh23eelyhnxvugw2nqq0q0dr6kws7q35jyqqnan33"
    export CHAIN_NAME="fantom"
    export VOTING_VERIFIER_CONTRACT_ADDRESS="axelar1elaymnd2epmfr498h2x9p2nezc4eklv95uv92u9csfs8wl75w7yqdc0h67"
    export MULTISIG_CONTRACT_ADDRESS="0x7eeE33A59Db27d762AA1Fa31b26efeE0dABa1132"
    ```
</tab-item>

<tab-item title="<code>devnet-amplifier</code>">
    ```bash
    export REWARDS_CONTRACT_ADDRESS="axelar1vaj9sfzc3z0gpel90wu4ljutncutv0wuhvvwfsh30rqxq422z89qnd989l"
    export CHAIN_NAME="fantom"
    export VOTING_VERIFIER_CONTRACT_ADDRESS="axelar1ty7mx0cllgz8xuvhn2j7e3qy8999ssgmjtktyqetl335em0g88lq6rjhl2"
    export MULTISIG_CONTRACT_ADDRESS="axelar19jxy26z0qnnspa45y5nru0l5rmy9d637z5km2ndjxthfxf5qaswst9290r"
    ```
</tab-item>

</tabs>

```bash
axelard q wasm contract-state smart REWARDS_CONTRACT_ADDRESS \

> '{

Reviewer: tabbing and variables

"rewards_pool":
{
"pool_id":
{
"chain_name":CHAIN_NAME,
"contract":VOTING_VERIFIER_CONTRACT_ADDRESS || MULTISIG_CONTRACT_ADDRESS
}
}
}'

> --from [key] \

Reviewer: key

--gas auto \
--gas-adjustment 2 \

> --node $RPC

Reviewer: variable?

Use `add_rewards()` add funds to a pool:

<tabs>

<tab-item title="<code>devnet-verifiers</code>">
    ```bash
    export REWARDS_CONTRACT_ADDRESS="axelar1guczj53xxl4347adagh23eelyhnxvugw2nqq0q0dr6kws7q35jyqqnan33"
    export CHAIN_NAME="fantom"
    export VOTING_VERIFIER_CONTRACT_ADDRESS="axelar1elaymnd2epmfr498h2x9p2nezc4eklv95uv92u9csfs8wl75w7yqdc0h67"
    export MULTISIG_CONTRACT_ADDRESS="0x7eeE33A59Db27d762AA1Fa31b26efeE0dABa1132"
    ```
</tab-item>

<tab-item title="<code>devnet-amplifier</code>">
    ```bash
    export REWARDS_CONTRACT_ADDRESS="axelar1vaj9sfzc3z0gpel90wu4ljutncutv0wuhvvwfsh30rqxq422z89qnd989l"
    export CHAIN_NAME="fantom"
    export VOTING_VERIFIER_CONTRACT_ADDRESS="axelar1ty7mx0cllgz8xuvhn2j7e3qy8999ssgmjtktyqetl335em0g88lq6rjhl2"
    export MULTISIG_CONTRACT_ADDRESS="axelar19jxy26z0qnnspa45y5nru0l5rmy9d637z5km2ndjxthfxf5qaswst9290r"
    ```
</tab-item>

</tabs>

```bash
axelard tx wasm execute REWARDS_CONTRACT_ADDRESS \
'{
"add_rewards":
{
"pool_id":
{
"chain_name":CHAIN_NAME,
"contract":"VOTING_VERIFIER_CONTRACT_ADDRESS || MULTISIG_CONTRACT_ADDRESS
}
}
}' \
--amount [amount to add] \
--from [key] \
--gas auto \
--gas-adjustment 2 \
--node $RPC
```

> Anyone can increase the number of funds in a rewards pool at any time.

Reviewer: ```suggestion
Anyone can add funds to a rewards pool at any time.
```
